### PR TITLE
Add file header snippet with support for several languages

### LINKED ISSUE #5: Seneca comment snippets
From Slack:

> It think one of the things that can be useful is automatic comment header to all of the files that will pre-populate Name, Seneca student Id, course code, etc. Usually, we do it for all programming assignments. This can also have a line about the copyright and code of conduct.

### PR DESCRIPTION
**Pull request by Myrfion**
# Description

Add a snippet for creating the header for files with information about the student and the file itself (create date, path) 

Fixes #5 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

I tested it manually way just by copying the project's directory and pasting it to the extensions folder in `~/.vscode/extensions` and reloading vs code after that. The newly created snippet `!fhead` should become available for you (refer to `package.json` for supported languages)

### CODE REVIEW

**Review by humphd — APPROVED** [review]
This is great!  For others reviewing and testing this, see https://code.visualstudio.com/docs/editor/userdefinedsnippets.

In follow-up PRs we can pull some of this info from data stored in the extension (when we have it).  For example: student details.

File: snippets/snippets.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "Student File Header Template": {
+    "prefix": "!fhead",
+    "body": [
+      "$LINE_COMMENT $RELATIVE_FILEPATH",
+      "$LINE_COMMENT Created on: $CURRENT_MONTH_NAME $CURRENT_DATE $CURRENT_YEAR",
+      "$LINE_COMMENT Student name: ${1:fullname}",
+      "$LINE_COMMENT Student email: ${2:email}@myseneca.ca",
+      "$LINE_COMMENT Class: ${3:classname}",
```
Comment (rudychung) [inline snippets/snippets.json:9]: Just a minor nitpick but I think the terminology of "Course" would be more appropriate "Class".

**Review by rudychung — COMMENTED** [review]
Looks good! I read up on how snippets in VSCode worked to review this, and this is really interesting. Just have one minor terminology issue with it.

File: snippets/snippets.json (hunk 1 above)
Comment (Myrfion) [reply snippets/snippets.json:9]: agree

Comment (Myrfion) [reply snippets/snippets.json]: updated

**Review by rudychung — APPROVED** [review]
Thanks for making the change.

**Pull request merged**